mewarpx/tests/test_particle_diag.py

```diff
@@ -0,0 +1,111 @@
+"""
+Test that verifies data is plotted correctly from the
+Monte-Carlo Collision script based on case 1 from
+Turner et al. (2013) - https://doi.org/10.1063/1.4775084
+"""
+
+import pytest
+import os.path
+import glob
+
+from mewarpx import util as mwxutil
+
+import numpy as np
+
+import collections
+import os
+import numpy as np
+import pandas
+import pytest
+
+from mewarpx import util as mwxutil
+
+def test_particle_diag():
+    dim = 2
+
+    # Initialize and import only when we know dimension
+    mwxutil.init_libwarpx(ndim=dim, rz=False)
+    from mewarpx import testing_util
+    from mewarpx.setups_store import diode_setup
+    from mewarpx.mwxrun import mwxrun
+
+    import mewarpx.mwxconstants as constants
+
+    post_proc_str = "_with_post_processing"
+    test_name = "particle_diag_test" + post_proc_str
```

Comment:
Such a nitpick but this could be one string since the name of the file doesn't depend on any parameters.